History.md

```diff
@@ -4,6 +4,8 @@ unreleased
   * Fix incorrect middleware execution with unanchored `RegExp`s
   * Fix `res.jsonp(obj, status)` deprecation message
   * Fix typo in `res.is` JSDoc
+  * deps: statuses@2.0.0
+    - Remove status[code] exports; use status.message[code]
```

Comment:
I don't believe this change is observable through Express, right? If so, can you show how a user of Express would notice this change in the dependency's functionality?

Reply:
I don't think a Express user will notice anything. Maybe a few will notice that 425 message renamed from `Unordered Collection` to `Too Early`, This possibility is very weak ..

Reply:
The change log is not composed of things that are most likely for people to notice, rather what is noticeable (an absolute). Typically it is most useful to list things that are less noticeable anyway, as those generally end up as the most surprising changes.